hooks/pre/30-upgrade.rb

```diff
@@ -164,6 +171,7 @@ def fail_and_exit(message)
     upgrade_step :mark_qpid_cert_for_update
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :remove_gutterball
+    upgrade_step :delete_event_queue
     upgrade_step :start_tomcat, :run_always => true
```

Comment:
@stbenjam after talking to @barnabycourt he suggests we create the queue all in this step in the hook. We also have the new PR's to make sure things are correct with just a normal installer run. Your thoughts?